modules/lvms-limitations-to-configure-size-of-devices.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="limitations-to-configure-size-of-devices_{context}"]
+= Limitations to configure the size of the devices to be used in LVM Storage
```

Comment:
```suggestion
= Limitations to configure the size of the devices used in LVM Storage
```

Reply:
To shorten this title, I think "to be" can be removed since you already use "used" here.

Reply:
Updated as per the suggestion.